### PR TITLE
initial submission

### PR DESCRIPTION
**Pull request by jhockert**
i probably missed a few things, please have a look

### CODE REVIEW

**Comment by jhockert** [comment]
Still figuring out github, excuse the spam

**Comment by jhockert** [comment]
@slawkens I copied your highscores-template from develop branch. From what I could see it wasn't working yet, is there any reason for me to look into it or have you already solved it?

**Comment by slawkens** [comment]
You also need to be running the develop branch of the site. Other than that, clear cache in admin panel or enable dev env mode in config.local.php

**Comment by jhockert** [comment]
Please have a look and see if you think it is ready

**Comment by slawkens** [comment]
Some things I noticed:
  - Wrong link name for Change Name option (saying 2x the same)
image
  - plugin.json is missing (I already commited it)

Other than that, very nice piece of code! You did a lot of work here.
I will definitely merge it.

**Comment by jhockert** [comment]
A bit of information since my last commit.. 
I have tried to rely on JavaScript where possible for minor design changes and by that managing to reduce the number of twig-templates. And the twig templates that are still there have been altered greatly. For my production server I tried to refrain from changing to much php-code so that it would be more suited for a wider audience. 

**Notable features i included** 
- Automatic generation of navbar logo
image
- config.php for customisable colouring, ability to change between dark and light mode (this is also called top from the JavaScript bits)

**Quirks I know of**
- Houses don't seem to show picture, this is because I generated house pictures from the map for every house and displayed them with a slider on my production server
- When viewing on smaller viewports the footer text colour does not change between width 1200px - 750px. Likely to some CSS mistake on my part
- From what I can remember when making this, some forms are not included in twig templates. This has resulted in some forms not getting the bootstrap treatment, for example admin view of new forums 
image
- Some headlines may not have correct colouring when configuring with config.php

It would be great to get some help with testing this template and ironing out issues. I did test it just now on a fresh install, and it seems to be quite ready..

I'd also like to point out that I'm a pure hobbyist when it comes to coding, so I'm sure some code isn't the best it could be.

**Comment by slawkens** [comment]
So, first what I noticed that you moved files into new directory.

They are not directlry into templates/ directory, which will not work.

They need to be in subfolder, like

templates/reborn/ (here the .twig files and all)

**Comment by jhockert** [comment]
My bad, should've had a coffee before I committed

File: reborn-template/templates/reborn/mail.account.register.html.twig [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
```
Comment (slawkens) [inline reborn-template/templates/reborn/mail.account.register.html.twig:10]: What's wrong with this file?

**Pull request merged**